devops/deploy/spin-down-script.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
+PORT_LIST="peer_list.txt"
```

Comment:
Given the file rename, would be good to also adjust the variable names and comments to match.